core/src/test/java/dk/alexandra/fresco/framework/network/AbstractCloseableNetworkTest.java

```diff
@@ -349,4 +349,58 @@ public void testConnectTimeout() {
     newCloseableNetwork(confs.get(0), Duration.ofMillis(10));
   }
 
+  @Test(timeout = TWO_MINUTE_TIMEOUT_MILLIS)
+  public void testAlternateReceivers() throws InterruptedException, ExecutionException {
+    alternateReceivers(10000);
+  }
+
+  private void alternateReceivers(int numMessages) throws InterruptedException, ExecutionException {
```

Comment:
Possibly add a comment about what this test is doing.

Reply:
Added